ship_functions.py

```diff
@@ -0,0 +1,212 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Aug 15 11:37:18 2019
+
+@author: j-bd
+
+"""
+
+import os
+import argparse
+import shutil
+
+from sklearn.model_selection import train_test_split
+import numpy as np
+import pandas as pd
+import skimage.io
+
+from mrcnn import utils
+from mrcnn import model
+from mrcnn import visualize
+import ship_config
+import ship_dataset
+
+
+def create_parser():
+    '''Get the informations from the operator'''
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-c", "--command", required=True,
+                        help="choice between 'train' and 'detection'")
+    parser.add_argument("-of", "--origin_folder", required=True,
+                        help="path to the Kaggle folder containing all data")
+    parser.add_argument("-pf", "--project_folder", required=True,
+                        help="path to your project folder")
+    parser.add_argument("-w", "--weights", required=True,
+                        help="Choice between 'coco', 'last', 'imagenet', or"\
+                        "the path to a weights file usedusable by mrcnn algorithm"\
+                        "to detect object")
+    parser.add_argument("-sr", "--split_rate", type=float, default=0.8,
+                        help="split rate between train and validation dataset during"\
+                        "mrcnn training")
+    args = parser.parse_args()
+    return args
+
+
+def structure(folder_list):
+    '''Create the structure for the project and downoald necessary file'''
+    for name in folder_list:
+        os.makedirs(name, exist_ok=True)
+
+
+def images_transfert(dataset, o_folder, f_folder):
+    '''Copy selected images from a folder to another one'''
+    filelist = list()
+    for image_name in dataset.ImageId.unique():
+        filelist.append(o_folder + image_name)
+    print(f"[INFO] Copying of {len(filelist)} images from {o_folder} to "\
```

Comment:
Instead of using syntax like `print("[INFO] blablabla)` you should consider using the `logging` base [module](https://docs.python.org/3.7/library/logging.html):
`logging.info('blablabla')`
`logging.warning('Careful !')`
etc...